src/main/java/archibald/Archibald.java

```diff
@@ -0,0 +1,206 @@
+package archibald;
+import java.util.Scanner;
+import java.io.*;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+
+import archibald.task.Deadline;
+import archibald.task.Event;
+import archibald.task.Task;
+import archibald.task.Todo;
+
+class ArchibaldException extends Exception {
+    public ArchibaldException(String message) {
+        super(message);
+    }
+}
+
+public class Archibald {
+
+    private static List<Task> tasks = new ArrayList<>();
+
+    public static void printArchibaldResponse(String message) {
+        System.out.println("____________________________________________________________");
+        System.out.println(message);
+        System.out.println("____________________________________________________________");
+    }
+
+    public static void addTask(String input) {
+        try {
+            String[] parts = input.split(" ", 2);
+            String type = parts[0].toLowerCase();
+    
+            if (type.equals("todo") || type.equals("deadline") || type.equals("event")) {
+                if (parts.length < 2 || parts[1].trim().isEmpty()) {
+                    throw new ArchibaldException("Error: By royal decree, the description of a " + type + " cannot be empty.");
+                }
+            }
+    
+            Task newTask;
+            switch (type) {
+                case "todo":
+                    newTask = new Todo(parts[1]);
+                    break;
+                case "deadline":
+                    String[] deadlineParts = parts[1].split(" /by ", 2);
+                    if (deadlineParts.length < 2) {
+                        throw new ArchibaldException("Error: Deadline must include '/by' followed by a date.");
+                    }
+                    newTask = new Deadline(deadlineParts[0], deadlineParts[1]);
+                    break;
+                case "event":
+                    String[] eventParts = parts[1].split(" /from | /to ");
+                    if (eventParts.length < 3) {
+                        throw new ArchibaldException("Error: Event must include '/from' and '/to' followed by dates.");
+                    }
+                    newTask = new Event(eventParts[0], eventParts[1], eventParts[2]);
+                    break;
+                default:
+                    throw new ArchibaldException("Error: In spite of my knowledge... I don't know what that means >:(");
```

Comment:
Based on requirements, there shouldn't be indentation in front of case clause. Please take note for all the switch below as well.
E.g.
```
switch (condition) {
case ABC:
    statements;
    // Fallthrough
case DEF:
    statements;
    break;
case XYZ:
    statements;
    break;
default:
    statements;
    break;
}
```